src/test/scala/tech/cryptonomic/conseil/tezos/PlatformDiscoveryOperationsTest.scala

```diff
@@ -27,6 +28,7 @@ class PlatformDiscoveryOperationsTest
   import slick.jdbc.PostgresProfile.api._
   import scala.concurrent.ExecutionContext.Implicits.global
 
+  val pdo = new PlatformDiscoveryOperations(ApiOperations(dbHandler))
```

Comment:
Trivial comment: I tend to use "system under test" (i.e. `sut`) as a conventionally defined name for the test subject.
Doing this by convention in situations like this, could remove ambiguity on what we're testing, and avoid the need to lookup what a new acronym stands for (`pdo` in this case)

Reply:
additional note: I didn't make the definition up myself, to be clear
https://en.wikipedia.org/wiki/System_under_test